rrtmgp/mo_gas_optics_rrtmgp.F90

```diff
@@ -646,8 +649,8 @@ function compute_gas_taus(this,                       &
             idx_h2o, col_dry_wk,col_gas, &
             fminor,jeta,tropo,jtemp,     & ! local input
             tau_rayleigh)
-      !$acc exit data detach(col_dry_wk) delete(this%krayl)
-      !$omp target exit data map(from:col_dry_wk) map(release:this%krayl)
+      !$acc exit data delete(this%krayl)
+      !$omp target exit data map(release:this%krayl)
```

Comment:
OK, good change. Getting rid of the `detach` on `col_dry_wk` should help with OpenMP v. OpenACC compatibility.